functests/install-script.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# This file is part of MinIO DirectPV
+# Copyright (c) 2022 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+echo "Installing direct PV"
+
+operating_system=""
+darwin="Darwin"
+linux="Linux"
+windows="Windows"
+
+function directpv(){
+    read operating_system kubectl_url minikube_url < <(initialize)
+    verify_kubectl
+    verify_krew 
+    kubectl krew install directpv
+    kubectl directpv install
+}
+
+# Fetch the operating system
+function initialize(){
+    case "$OSTYPE" in
+        darwin*)  
+            operating_system=$darwin            
+            kubectl_url="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
+            minikube_url="https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64"
+            ;;
+        linux*)   
+            operating_system=$linux
+            kubectl_url="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+            minikube_url="https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64"
+            ;;
+        msys*)    
```

Comment:
Are you working with `msys2` on MS Windows or are we supporting https://itsfoss.com/install-bash-on-windows/?